packages/react-pose/src/components/Transition/PoseGroup.tsx

```diff
@@ -3,14 +3,12 @@ import { Props } from './types';
 import Transition from './';
 
 // TODO: Deprecate, add notice about Transition and flipMove: true opt-in
-export default class PoseGroup extends React.PureComponent<Props> {
+export default class PoseGroup extends React.Component<Props> {
   static defaultProps = {
     flipMove: true
   };
 
   render() {
-    const { children, ...props } = this.props;
-
-    return <Transition {...props}>{children}</Transition>;
+    return <Transition {...this.props} />;
```

Comment:
children is a prop, no need extract it from `this.props` by making a temporary/intermediate `props` object